### DISCUSSION TITLE
Markdown editor controls should stay visible

### DISCUSSION THREAD

**Discussion by Christilut** [directus/directus#11125] [Feature Requests]
Not sure where I was supposed to post this, please change it if it's in the wrong place.

So I was typing a long piece of text (API documentation) and the markdown editor header (that has the controls) stays all the way at the top of the text. So I have to keep scrolling way up to use it. It would be nice if the header became fixed as soon as it would otherwise leave the screen but you're still editing the markdown.

**Comment by benhaynes** [comment]
Agree, same for other interfaces, like wysiwyg.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
